sdk/monitor/Microsoft.Azure.Management.Monitor/src/Microsoft.Azure.Management.Monitor.csproj

```diff
@@ -6,13 +6,11 @@
   <PropertyGroup>
     <Description>Microsoft Azure Monitor Library</Description>
     <AssemblyName>Microsoft.Azure.Management.Monitor</AssemblyName>
-    <Version>0.25.4-preview</Version>
```

Comment:
@HarveyLink @archerzz While I added this change as it was earlier requested in this review, I see that there have been multiple breaking changes in the past which did not bump up the major version to 1. Would it suffice if I bumped up the version to 0.26.0 (instead of 1.0.0)?

Reply:
Since it is a sider update rather than a major api version change, I agreed with your minor version 0.26.0-preview. @archerzz What's your opinion?

Reply:
@ritwik8119  I'm fine. Also if your next version is still `-preview`, then you're free to use `0.26.0-preview` since it's not official release. But you need mention this to the team for their awareness that we should avoid breaking changes after GA. Thanks.